tools/x-workbench/register-components.js

```diff
@@ -1,4 +1,5 @@
 const components = [
+	require('../../components/x-article-save-button/stories'),
```

Comment:
this should be `@financial-times/x-article-save-button` as with the rest of the components. adding it as a `file:` reference in `x-workbench`'s `package.json` will work even before the component is released